components/Releases/CreateRelease.jsx

```diff
@@ -17,7 +17,7 @@ const releaseTypes = [
   { id: 6, text: "Choose release type", isDisabled: true },
 ]
 
-export default function CreateRelease() {
+export default function CreateRelease({ setUpdateReleases }) {
```

Comment:
Change this to some that represents what is happening, like `onReleaseCreated`

Reply:
this is the same as the one you called out above. Will change.